src/pages/Main/MainCarousel/MainCarousel.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const MainCarousel = () => {
+  const settings = {
+    dots: false,
+    infinite: true,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: true,
+    autoplay: true,
+    autoplaySpeed: 3000,
+    pauseOnHover: true,
+    nextArrow: (
+      <NextTo>
+        <Img src="./images/Main/arrow-right.png" alt="right" />
+      </NextTo>
+    ),
+    prevArrow: (
+      <Pre>
```

Comment:
```suggestion
      <PrevBtn>
```
이게 컴포넌트의 이름으로 더 어울릴 것 같네요!